.gitignore

```diff
@@ -3,3 +3,5 @@
 coverage
 node_modules
 private-key.pem
+.idea/
```

Comment:
imho this should be a part of your own global gitignore rather than a part of the repository (a good reasoning behind it is written at https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/)

Reply:
Thanks for the link, its really nice write-up. Honestly, I'm used to adding these everywhere, since its few extra words in one file, vs constant dealing with someone committing IDE files. I did not even notice I have it in the PR.